source/features/show-whitespace.tsx

```diff
@@ -1,67 +1,20 @@
 import './show-whitespace.css';
-import React from 'dom-chef';
 import select from 'select-dom';
 import delegate from 'delegate-it';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
-import getTextNodes from '../helpers/get-text-nodes';
 import onNewComments from '../github-events/on-new-comments';
 import {onDiffFileLoad} from '../github-events/on-fragment-load';
 import {codeElementsSelector} from '../github-helpers/dom-formatters';
-
-// `splitText` is used before and after each whitespace group so a new whitespace-only text node is created. This new node is then wrapped in a <span>
-function showWhiteSpacesOn(line: Element): void {
-	const shouldAvoidSurroundingSpaces = Boolean(line.closest('.blob-wrapper-embedded')); // #2285
```

Comment:
`showWhiteSpacesOn` was moved without change, except for extracting `shouldAvoidSurroundingSpaces`